Source/WebKit/Platform/IPC/win/ArgumentCodersWin.cpp

```diff
@@ -34,7 +34,7 @@ namespace IPC {
 
 void ArgumentCoder<Win32Handle>::encode(Encoder& encoder, const Win32Handle& handle)
 {
-    encoder << handle.copy();
+    encoder << Win32Handle { handle };
```

Comment:
In subsequent patch you could change this to rvalue variant like in MachSengRight

Reply:
Sure! Any changes to generators required?